packages/frontend/store/category/actions.ts

```diff
@@ -1,10 +1,11 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
-import type { Http } from '@vse-bude/shared';
+import type { Http, HttpAcceptLanguage } from '@vse-bude/shared';
 import { getAllCategories, getAllCategoriesSSR } from 'services/category';
 import { CategoryActions } from './action-types';
 
 interface RequestOptions {
   limit?: number;
+  locale?: HttpAcceptLanguage;
```

Comment:
You don't need to pass it.

Reply:
how to get localized data?
should the translation be at the front?